Author: Milan P. StaniÄ‡ <mps@arvanta.net>
Date:   2021-04-13 06:32:07.000000000 +0000

musl doesn't support __free_fn_t, fix this by 'void (*)(void *)'

--- a//user/v9/drbdsetup_events2.c	2021-04-13 06:32:07.000000000 +0000
+++ b//user/v9/drbdsetup_events2.c	2021-06-23 06:52:47.451319822 +0000
@@ -1375,7 +1375,7 @@ void events2_prepare_update()
 /* Drop all data and start again with new initial state. */
 void events2_reset()
 {
-	tdestroy(all_resources, (__free_fn_t) free_resource);
+	tdestroy(all_resources, (void (*)(void *)) free_resource);
 	all_resources = NULL;
 	initial_state = true;
 }
